node/src/chain_spec.rs

```diff
@@ -300,3 +348,40 @@ fn statemine_testnet_genesis(
 		cumulus_pallet_aura_ext: Default::default(),
 	}
 }
+
+fn statemine_mainnet_genesis(
+	invulnerables: Vec<(AccountId, AuraId)>,
+	endowed_accounts: Vec<AccountId>,
+	id: ParaId,
+) -> statemine_runtime::GenesisConfig {
+	statemine_runtime::GenesisConfig {
+		frame_system: statemine_runtime::SystemConfig {
+			code: statemine_runtime::WASM_BINARY
+				.expect("WASM binary was not build, please build it!")
+				.to_vec(),
+			changes_trie_config: Default::default(),
+		},
+		pallet_balances: statemine_runtime::BalancesConfig {
+			balances: endowed_accounts
```

Comment:
When I do not endow the "Invulnerable account" with an "ED" the node crashes because the account does not exist. I am not sure exactly if giving the first invulnerable an ED is the best way to handle this but I am also not sure how to do this @gavofyork